test/examples/ucp_hello_world.c

```diff
@@ -379,8 +391,19 @@ static void barrier(int oob_sock)
     recv(oob_sock, &dummy, sizeof(dummy), 0);
 }
 
+static void generate_random_string(char *str, int size)
+{
+    int i;
+    srand(time(NULL)); // randomize seed
+    for( i = 0 ; i < size ; ++i) {
```

Comment:
Code style. Spaces are in wrong places.
Should be `for (i = 0; i < size; ++i) {`